ignore build products

Copyright (C) 2014 Curt Brune <curt@cumulusnetworks.com>

SPDX-License-Identifier:     GPL-2.0

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..ca3b55f
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,25 @@
+Doxyfile
+Makefile
+auto/Makefile
+auto/desc
+auto/types
+config.h
+config.log
+config.status
+libpopt.la
+libtool
+m4/Makefile
+po/Makefile
+po/Makefile.in
+po/POTFILES
+popt.lo
+popt.pc
+popt.spec
+poptconfig.lo
+popthelp.lo
+poptint.lo
+poptparse.lo
+stamp-h1
+tdict
+test1
+test2
